vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/TextArea.java

```diff
@@ -396,6 +396,36 @@ public void setPreventInvalidInput(boolean preventInvalidInput) {
         super.setPreventInvalidInput(preventInvalidInput);
     }
 
+    /**
+     * Sets a regular expression for the value to pass on the client-side. The
+     * pattern must be a valid JavaScript Regular Expression that matches the
+     * entire value, not just some subset.
+     *
+     * @param pattern
+     *            the new String pattern
+     *
+     * @see <a href=
+     *      "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefpattern">
+     *      https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefpattern</>
+     * @see <a href=
+     *      "https://html.spec.whatwg.org/multipage/input.html#attr-input-pattern">
+     *      https://html.spec.whatwg.org/multipage/input.html#attr-input-pattern</>
+     */
+    public void setPattern(String pattern) {
+        getElement().setProperty("pattern", pattern == null ? "" : pattern);
```

Comment:
Why not set the `null` value to the element? With the suggested approach:

```java
TextArea textArea = new TextArea();
textArea.getPattern(); // null
textArea.setPattern(null);
textArea.getPattern(); // ""
```

Reply:
You're right.